yb-voyager/src/dbzm/dbzm.go

```diff
@@ -46,6 +47,10 @@ func (d *Debezium) Start() error {
 	cmdStr := fmt.Sprintf("cd %q; %s > %s 2>&1", DEBEZIUM_DIST_DIR, filepath.Join(DEBEZIUM_DIST_DIR, "run.sh"), logFile)
 	log.Infof("running command: %s\n", cmdStr)
 	d.cmd = exec.Command("/bin/bash", "-c", cmdStr)
+	d.cmd.SysProcAttr = &syscall.SysProcAttr{
+		Pdeathsig: syscall.SIGKILL, // kill the debezium process if the parent process dies
```

Comment:
@sanyamsinghal just test this once on mac because I was getting an error - ` unknown field Pdeathsig in struct literal of type syscall.SysProcAttr`

Seems like maybe it's not supported on mac? - https://github.com/tebeka/selenium/issues/48